immuni_common/helpers/sanic.py

```diff
@@ -178,3 +184,52 @@ def _validate_json_content_type(request: Request) -> None:
         return _wrapper
 
     return _decorator
+
+
+async def wait_configured_time() -> None:
```

Comment:
Mark it as private if it ain't used anywhere else. 🙏

Reply:
I kinda recall in ingestion this function is used by the `slow_down` decorator too. In that case, wouldn't it be better to pass the `timeout` and `sigma` as explicit arguments?

Reply:
It is also used there but it doesn't use different parameters, as long as it's the same configuration it should be fine.

Reply:
Then a question: why a generic `slow_down` decorator would take a specific `DUMMY_REQUEST_TIMEOUT_MILLIS` config? If the `slow_down` logic is only for dummy requests, shouldn't it be part of this decorator too? 🤔

Reply:
The `slow_down` logic in the ingestion server is not for dummy requests, in that case it's explicitly to slow down any attempts at brute-forcing a valid OTP code (and it's applied in any case, dummy or not)

Reply:
Okay, then do you agree that having it using `DUMMY_REQUEST_TIMEOUT_MILLIS` in the OTP context is somewhat off? 🤔

Reply:
Yup, we can introduce a new variable when integrating this change there as well.